### DISCUSSION TITLE
Multiple key mod

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#14914] [Ideas]
Allows users to use any alphabet key to click circles. This would allow people to play 500bpm streams and crazy patterns.

**Comment by smoogipoo** [comment]
It doesn't work like that, but you can bind two keys for each action in settings already.

**Reply by peppy** [reply]
I think they are asking for a mod where you can use any key on the keyboard to hit. Sounds maybe fun but might be a bit of a pain to implement well.

**Reply by developomp** [reply]
The "Alphabet keys" might have some edge cases now that I think about it. How about taking any keystroke as input unless it is a function key (esc, tab, F1~F12, etc.)?

**Reply by goodtrailer** [reply]
Maybe there should be a limit like 7? I mean, 7 keys is more than enough to hit 500 bpm streams. I can't really imagine how you'd play with more than even 5. Or maybe not, since having a limit goes against the "no holds barred" idea.

just saying, if the limit were 7 keys, you could totally call the mod "7k" and reuse the mod icon from mania if it ever gets one

**Reply by developomp** [reply]
Not disagreeing with you or anything but why the number 7?

**Reply by goodtrailer** [reply]
5 fingers on the left hand plus the mouse buttons. If you really want to get creative, I guess you could slide your hand across the keyboard, then you could hit some real freaky patterns, in which case there should be no limit.
